scripts/kconfig/kconfigfunctions.py

```diff
@@ -375,6 +375,21 @@ def dt_node_reg(kconf, name, path, index=0, unit=None):
     if name == "dt_node_reg_addr_hex":
         return hex(_dt_node_reg_addr(kconf, path, index, unit))
 
+def dt_nodelabel_reg(kconf, name, label, index=0, unit=None):
+    """
+    This function just routes to the proper function and converts
+    the result to either a string int or string hex value.
```

Comment:
Please improve this to say what it actually does

Reply:
Copied the style of several other `_nodelabel` variants